projects/core/src/lib/datatable/table/table.component.ts

```diff
@@ -243,16 +244,30 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
         this.dialog.open(FullDisplayComponent, {data: {xStrings: coords[0], yStrings: coords[1], geometry: output.getGeometry()}});
     }
 
-    readTimePropertyStart(geometry: OlFeature): string {
-        const minimum = '-262144-01-01T00:00:00+00:00';
+    readTimePropertyStart(geometry: OlFeature): string | undefined {
         const result = geometry['values_']['_____table__start'];
-        return result === minimum ? '-∞' : result;
+        if (!result) {
+            return undefined;
+        }
+        try {
+            const timeInstant = new Time(result);
+            return timeInstant.startStringOrNegInf();
+        } catch (e) {
+            return result;
+        }
```

Comment:
Why do we need the try-catch here? It doesn't seem that this can fail.

Reply:
i dont know what the table does so wanted to be safe

[b4be2d0](https://github.com/geo-engine/geoengine-ui/pull/532/commits/b4be2d041e74c331cf5ef831da0c86538d03a146)